documentation/source/quickstart.rst

```diff
@@ -284,8 +286,16 @@ We can also cast the signal handle directly to an integer:
 
 
 
-Parallel and sequential execution of coroutines
```

Comment:
"coroutines" haven't been introduced yet, and I feel a heading is not the right place to do it.